async/kafka/simple/simple.go

```diff
@@ -171,3 +172,12 @@ func closePartitionConsumer(cns sarama.PartitionConsumer) {
 		log.Errorf("failed to close partition consumer: %v", err)
 	}
 }
+
+func containsEmtpyValue(values []string) bool {
```

Comment:
we could introduce a common function to handle both simple and group.maybe in the parent package or in a new strings package...

Reply:
Yeah, good point!

I was thinking about it. That's actually the reason why I duplicated this. 
- I didn't want to move it to the parent package because in that case, it would be public which might be weird from the API perspective. I think that having something like `IsSliceEmpty` here https://godoc.org/github.com/beatlabs/patron/async/kafka would be redundant.
- It would be an option to have an internal package for purposes like this ( common validation, helpers, etc. )

Would you agree with an internal package? Should we create a common purpose one or per case package?

WDYT?

Reply:
helpers is too generic. validation makes sense.

Reply:
Updated. Added `validation` package and moved validation. 
PTAL.